server/api-manager.js

```diff
@@ -936,16 +938,14 @@ export class MindbodyQueries {
     return false;
   }
 }
+
 const MindbodyAccess = new MindbodyQueries();
 export default MindbodyAccess;
 
 /*example code
 var client = new mindbodyQueries();
 client.getAuth()
-  .then((value) => {
-    client.authToken = value.AccessToken;
-    return client.getClients();
-  })
+  .getClients()
   .then((value) => {
     console.log(value);
   })
```

Comment:
I know this is a comment full of sample code, but I do need this stuff to figure out how to use Promises and the library. So, I really want to get this thing right.
 - First, let's rename var client because it's an ambiguous name. Is this the "client" of the MB server? Is it related to the getClients call?
 - I also think that .getClients needs to happen in a getAuth().then() function. I could be wrong about that.

```
/*example code
var api = new mindbodyQueries();
api.getAuth()
  .then(api.getClients())
  .then((clientList) => {
    console.log(clientList);
  });
```

Reply:
I agree this sample code should be correct for learning purposes. You are correct that getClients() needs to be inside of the getAuth().then() but it also needs to be returned. I tested this example code and it does return the appropriate information about the clients.

/*example code
var api = new MindbodyQueries();
api.getAuth()
  .then(() => {
    return api.getClients();
  })
  .then((clientsList) => {
    console.log(clientsList);
  })
  .catch((m) => {
    console.log(m);
  });
  */

Sorry for the terrible formatting I couldn't figure out how to make it look better.

Reply:
Thanks. These are all improvements. Thanks for fixing the capitalization of mindbodyQueries... I just copy-and-pasted that :(. 

- Now that I'm thinking about it, users of this library won't do "var api = new MindbodyQueries();" They should just reference the MindbodyAccess singleton object.

- also, the catch block is good for the example. We should have those wherever we use MBAccess let's rename it to "error" and call  error.toString() because I think it's possible for any Promise in the chain to return non-string objects that won't display well with a straightforward console.log.

How about:

`
/*example code:
import MindbodyAccess from "../../api-manager.js";
//...
  MindbodyAccess.getAuth()
  .then(() => {
    return MindbodyAccess.getClients();
  })
  .then((clientsList) => {
    console.log(clientsList);
  })
  .catch((error) => {
    console.log("Error occurred in [name the context]: " + error.toString());
  });
*/
`
^^^ FYI To make this code block, I'm trying to use the formatting tool with a "<>" icon in the git web client.

Reply:
Looks good!